src/qoi/include/grins/qoi_base.h

```diff
@@ -78,7 +78,9 @@ namespace GRINS
      * Method to allow QoI to cache any system information needed for QoI calculation,
      * for example, solution variable indices.
      */
-    virtual void init( const GetPot& input, const MultiphysicsSystem& system );
+    virtual void init( const GetPot& input,
+                       const MultiphysicsSystem& system,
+                       unsigned int qoi_num );
```

Comment:
Also?  We totally need to add libmesh_move, libmesh_final, libmesh_override, etc. for all the functions and keywords where C++11 gives you more efficiency or better error checking but where C++98 still compiles to correct code with a no-op function or an empty macro.